data.js

```diff
@@ -6,28 +6,35 @@ class Data {
     this.x = x;
     this.y = y;
     this.color = palette.FG; //TODO: pass this in as arg rather than global variable?
+    this.alpha = 255;
     this.sz = baseSize;
     this.refined = false;
     this.binIt = false;
     this.bin = undefined;
+    this.binPauseTime = 30;
```

Comment:
This variable can be tweaked (or removed) to pause for a shorter period before sending digit to bin